samsung_mdc/fields.py

```diff
@@ -151,3 +151,16 @@ def pack(self, data):
         if not len(rv) == 4 or not all(0 <= x < 256 for x in rv):
             raise ValueError('Invalid IP address', data)
         return rv
+
+
+class VideoWall(Field):
```

Comment:
I think I need a new field for this
![Video Wall Model def](https://user-images.githubusercontent.com/13696924/142895385-d3db735a-939f-4d80-a27a-5066d60948ff.png)
: